openmetadata-ui/src/main/resources/ui/src/components/DatasetDetails/datasetDetails.style.less

```diff
@@ -18,5 +18,5 @@
   flex-grow: 1;
   padding: 1rem /* 16px */;
   box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
-  border-radius: 0.375rem /* 6px */;
+  border-radius: @border-radius;
```

Comment:
Variable AntD has is
```
@border-radius-base
@border-radius-sm
```

Did you add this variable? I feel our base value is already 4px can we use it?